src/Spec/Swagger2.php

```diff
@@ -169,7 +171,8 @@ public function getMethods($service)
                         'packaging' => $method['x-appwrite']['packaging'] ?? false,
                         'title' => $method['summary'] ?? '',
                         'description' => $method['description'] ?? '',
-                        'security' => [$auth] ?? [],
+                        'auth' => [$methodAuth] ?? [],
+                        'security' => [$methodSecurity] ?? [],
```

Comment:
we are adding auth and security, are we using `security` for anything?

Reply:
To clarify: we already had `security`, but it was using `auth` from the spec. I have added `auth` and made this use `auth` from the spec, and changed the existing `security` to use `security` from the spec.

We are only using `security` in the HTTP examples here: https://github.com/appwrite/sdk-generator/blob/969bfa308f9504b7aaa45fa064f9e2ca83858343/templates/http/docs/example.md.twig#L10

Reply:
Allright.